singularity-eos/closure/mixed_cell_models.hpp

```diff
@@ -182,6 +182,7 @@ class PTESolverBase {
   void Finalize() {
     for (int m = 0; m < nmat; m++) {
       temp[m] *= Tnorm;
+      PORTABLE_REQUIRE(temp[m] > 0., "Non-positive temperature returned");
```

Comment:
Should exactly zero temperature be supported? I.e., `temp[m] >= 0`?

Reply:
Good point... I think I agree with supporting zero temperature